client/src/Block/components/BlockAuthor.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react'
+import { Link } from 'react-router-dom'
+
+// common
+import { shortString } from 'common/helpers'
+import { INTERNAL_ROUTES } from 'common/routes'
+
+type Props = {
+  isDesktop: boolean
+  chain: string
+  author?: string
+}
+
+const BlockAuthor: FC<Props> = ({ author, isDesktop, chain }) => {
+  const hasBlockAuthor = author !== undefined && author !== null
+  const blockAuthor = author || 'Unknown'
```

Comment:
it seems like it will never render the `"Uknown"` author because if `hasBlockAuthor` condition is not met it will render `<div>Unknown</div>`